components/cli/pkg/artifacts/apim.go

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2019 WSO2 Inc. (http:www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http:www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package artifacts
+
+import (
+	"path/filepath"
+
+	"github.com/cellery-io/sdk/components/cli/pkg/kubectl"
+)
+
+func CreateGlobalGatewayConfigMaps() error {
+	for _, confMap := range buildGlobalGatewayConfigMapPathsWithConfigName() {
+		err := kubectl.CreateConfigMapWithNamespace(confMap[0], confMap[1], "cellery-system")
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func buildGlobalGatewayConfigMapPathsWithConfigName() [][]string {
```

Comment:
You can return an array of `ConfigMap` instead of a 2d string array

Reply:
+1, will update